web/src/pages/people/index.tsx

```diff
@@ -72,16 +106,40 @@ export default function PeoplePage() {
         }
       />
 
-      {!isTreOpsStaff && (
-        <div className={styles["button-container"]}>
-          {isAdmin && <ApprovedResearcherImport />}
-          {(isAdmin || isIAO) && <ExternalInvite />}
-        </div>
-      )}
+      <div className={styles["button-container"]}>
+        {(isAdmin || isTreOpsStaff) && (
+          <form className={styles["search-container"]}>
+            <Input
+              placeholder="search users..."
+              id={styles.search}
+              name="search"
+              onChange={handleInputChange}
+              value={searchTerm}
+              aria-label="search users of the portal"
+            ></Input>
```

Comment:
Would some help text here be beneficial so the user knows what terms they can search by?

Reply:
https://github.com/ucl-arc-tre/portal/pull/251/commits/045cc2464802aba82c28db257ec209f4a65cd326